sort.c

```diff
@@ -0,0 +1,28 @@
+// This program will sort the Strings in the Lexicographic order
+#include<stdio.h>
+#include<stdlib.h>
+#include<string.h>
+
+void sort(char* str)
+{
+    int number=0,i,j;
+    char temp;
+    int l=strlen(str);

```

Comment:
make changing in all the files with correct naming conventoons, spacing... follow the coding conventions for c. and let me know once you do that.